changelogs/unreleased/gh-7886-dont-close-xlog-fd-atfork.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed a bug when Tarantool could execute random bytes as a Lua code after fork
+  (gh-7886).
```

Comment:
Nit: I'd explicitly mention in the changeog that it was only possible with glibc <= 2.28.

Reply:
Done